crates/rolldown/src/utils/chunk/deconflict_chunk_symbols.rs

```diff
@@ -99,9 +100,18 @@ pub fn deconflict_chunk_symbols(
   if matches!(format, OutputFormat::Esm) {
     chunk.imports_from_external_modules.iter().for_each(|(module, _)| {
       let db = link_output.symbol_db.local_db(*module);
-      db.classic_data.iter_enumerated().skip(1).for_each(|(symbol, _)| {
-        renamer.add_symbol_in_root_scope((*module, symbol).into());
```

Comment:
I am not sure it is a heck or somethingelse, for this case: 


```js
import * as b from 'b'

console.log(`b: `, b)
```
If you remove `skip(1)` it will generate output: 

```js
import * as b$1 from "b";

//#region entry.js
console.log(`b: `, b$1);

//#endregion
//# sourceMappingURL=entry.js.map
```

Reply:
Codes are introduced in this PR https://github.com/rolldown/rolldown/pull/3082/files#diff-3e1292ddb548179d38ab0c4ce82cc9a16844334560d9e9c8ef26ebab2b389899R92-R99.

I looked up the PR again and didn't found what does the `skip(1)` do.

Reply:
It is used to filter unused external symbol, which avoid same symbol name to be added twice, once for external symbol, once for import_stmt declared symbol.